fuse/core.go

```diff
@@ -55,12 +54,12 @@ var uid, gid = getIDs()
 
 type fuseNode struct {
 	ftype             string
-	parent            plugin.Parent
+	parent            *dir
 	entry             plugin.Entry
```

Comment:
I still need to store name and id. I guess I could do that directly.

Reply:
There's several reasons that doesn't work. Using it to determine whether it's a directory, fallback in case this is a plugin root.

Reply:
Ah that sucks. Could we use ftype to store all of that when the fuseNode's created?

Reply:
I'll work on it after lunch.

Reply:
Too much of the system is built on having an entry. It may not always be what we want, but often it is. I could only get rid of the entry reference if I moved to always recreating the entire history because I changed to `parent *dir`, so `dir` needs to have an entry.

Reply:
I do have a few small tweaks to add though.

Reply:
Ok, I don't have an issue with the current changes too. Thanks for digging into that.